analyzers/tests/SonarAnalyzer.UnitTest/SymbolicExecution/Roslyn/RoslynSymbolicExecutionTest.Invocation.cs

```diff
@@ -18,6 +18,7 @@
  * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
  */
 
+using System.Security.AccessControl;
```

Comment:
?

Reply:
WTF? VS is recently adding totally nonsence namespaces

Reply:
I experienced the same a couple of times